src/libs/analytics/provider.ts

```diff
@@ -28,6 +29,15 @@ export const analytics: AnalyticsProvider = {
     if (eventName.amplitude) {
       amplitude.logEvent(eventName.amplitude, params)
     }
+    if (await getIsMaestro()) {
+      await fetch('http://10.0.2.2:4001', {
```

Comment:
Sonar remonte un problème de sécurité due à la non utilisation de HTTPS. Je suppose qu'on veux ignorer cette issue ? Si c'est le cas, est-ce que tu peux ajouter `// NOSONAR + le pourquoi de l'ignore` comme ceci et voir si ça ignore la règle 
```suggestion
      await fetch('http://10.0.2.2:4001', { // NOSONAR(typescript:S5332) + commentaire
```

Reply:
c'est fait =)